lib/basket.rb

```diff
@@ -0,0 +1,6 @@
+class Basket
```

Comment:
Is this the start of your abstraction from the 'god class'?

Reply:
it was!